source/sass/buyers-guide/views/category.scss

```diff
@@ -9,55 +9,31 @@
     max-width: 800px;
   }
 
-  .categories-container {
-    display: flex;
-    flex-wrap: wrap;
+  .category-item-container {
+    $padding-x: 12px;
+    $padding-y: 25px;
 
-    .category-item-container {
-      $padding-x: 12px;
-      $padding-y: 12px;
-
-      position: relative;
-      width: 50%;
-      text-align: left;
-
-      &:nth-child(even) {
```

Comment:
Removing this creates the only visual deviation from the current site. The current site has a total of ~40px between products whereas this update has 30px. I referred back to the mock and it has 30px between products, but the live site has more. Feedback on how to move forward with this is more than welcomed.

Reply:
In this case, removing the 3% padding CSS rules is def a great move. 

Our live site, redpen/sketch file, and ticket specs aren't always up-to-date so I usually check with the designers to verify what the expected design is. We as a team should really get better at this though so we can reduce the overhead we spend on just finding "what specs do I follow". Code and design cleanup for the Foundation site is ongoing so hopefully this kind of issues will go away soon.

Reply:
Awesome :thumbsup: